figma-import.config.js.example

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  FIGMA_TOKEN: '',
+  FILE_KEY: '',
+  FRAME_WITH_ICONS_ID: '',
+  CONCURRENCY: 1,
+}
```

Comment:
This file needs some documentation, I have no idea what this options are supposed to be supplied with